packages/pipeline-editor/style/index.css

```diff
@@ -179,3 +179,8 @@ span.elyra-BreadCrumbs-disabled svg path.jp-icon3[fill] {
 span.elyra-BreadCrumbs-disabled + span {
   color: var(--md-grey-500);
 }
+
+input.elyra-Dialog-checkbox {
```

Comment:
nit: why capitalize Dialog here?

Reply:
I just followed what lab uses for dialogs, like `.jp-Dialog-content`